external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -1,26 +1,27 @@
 if(${ENABLE_PCMSolver})
-    find_package(PCMSolver QUIET)  # todo CONFIG
+    find_package(PCMSolver CONFIG QUIET)
 
     if(${PCMSolver_FOUND})
-        #get_property(_loc TARGET PCMSolver::pcm PROPERTY LOCATION)  # todo
-        get_property(_loc TARGET PCMSolver::PCMSolver PROPERTY LOCATION)
+        get_property(_loc TARGET PCMSolver::pcm PROPERTY LOCATION)
         message(STATUS "${Cyan}Found PCMSolver${ColourReset}: ${_loc} (found version ${PCMSolver_VERSION})")
         add_library(pcmsolver_external INTERFACE)  # dummy
     else()
         include(ExternalProject)
         message(STATUS "Suitable PCMSolver could not be located, ${Magenta}Building PCMSolver${ColourReset} instead.")
 
         if(${BUILD_SHARED_LIBS})
```

Comment:
What is the suggested way of building the add-ons? As static or as shared libraries? The former **does not** currently work with PCMSolver due to PCMSolver/pcmsolver#34 (yes, I am a moron...)